OpenDreamShared/Dream/Procs/DreamProcOpcode.cs

```diff
@@ -248,4 +249,31 @@ public enum Type : byte {
             }
         }
     }
+
+    // Dummy class-as-namespace because C# just kinda be like this
+    public static class OpcodeVerifier
+    {
+        /// <summary>
+        /// Validates that the opcodes in DreamProcOpcode are all unique, such that none resolve to the same byte.
+        /// </summary>
+        /// <returns>True if there are duplicate opcodes, false if not</returns>
+        public static bool AreOpcodesInvalid() // FIXME: Can this be made into something done during compiletime? Like, *this code's* compiletime? >:/
+        {
+            // I'm not *too* satisfied with this boolean schtick, as opposed to throwing,
+            // but since we're in OpenDreamShared I want each executable to be able to do what they want with this information.
+            
+            ///Key is an int we're already using for an opcode
+            HashSet<int> byte_pool = new(); // FIXME: Do not assume that the underlying type is an int. Somehow. Y'all don't have constexpr so
+            foreach (int used_int in Enum.GetValues(typeof(DreamProcOpcode)))
+            {
+                if(byte_pool.Contains(used_int))
+                {
+                    return true;
+                }
+                byte_pool.Add(used_int);
+            }
```

Comment:
You can use `DreamProcOpcode` instead of `int` to do what the `FIXME` suggests.

`byte_pool` and `used_int` do not fit the naming convention for local variables in the rest of the project.
```suggestion
            HashSet<DreamProcOpcode> bytePool = new();
            foreach (DreamProcOpcode usedInt in Enum.GetValues(typeof(DreamProcOpcode)))
            {
                if(bytePool.Contains(usedInt))
                {
                    return true;
                }
                bytePool.Add(usedInt);
            }
```